src/channels/components/ChannelForm/ChannelForm.tsx

```diff
@@ -79,91 +72,62 @@ export const ChannelForm: React.FC<ChannelFormProps> = ({
   onMarkAsPaidStrategyChange,
 }) => {
   const intl = useIntl();
-  const [copied, copy] = useClipboard();
+  const [, copy] = useClipboard();
   const formErrors = getFormErrors<keyof FormData, ChannelErrorFragment>(
-    ["name", "slug", "currencyCode", "defaultCountry"],
+    [
+      "name",
+      "slug",
+      "currencyCode",
+      "defaultCountry",
+      "deleteExpiredOrdersAfter",
+    ],
     errors,
   );
-  const classes = useStyles();
 
   return (
     <>
-      <Card>
-        <CardTitle
-          title={intl.formatMessage(commonMessages.generalInformations)}
-        />
-        <CardContent>
-          <TextField
+      <DashboardCard>
+        <DashboardCard.Title>
+          {intl.formatMessage(commonMessages.generalInformations)}
+        </DashboardCard.Title>
+        <DashboardCard.Content>
+          <Input
             error={!!formErrors.name}
             helperText={getChannelsErrorMessage(formErrors?.name, intl)}
             disabled={disabled}
-            fullWidth
-            label={intl.formatMessage({
-              id: "UymotP",
-              defaultMessage: "Channel name",
-              description: "channel name",
-            })}
+            label={intl.formatMessage(messages.channelName)}
             name="name"
             value={data.name}
             onChange={onChange}
           />
           <FormSpacer />
-          <TextField
+          <Input
             error={!!formErrors.slug}
             helperText={getChannelsErrorMessage(formErrors?.slug, intl)}
             disabled={disabled}
```

Comment:
issue: since country selector is still using old macaw component, please set for now the same height
```
__height={48.44}
```

it's hacky, but we can keep that within the dashbaord for a while